src/svd/interrupt.rs

```diff
@@ -0,0 +1,50 @@
+use crate::chip;
+use crate::DisplayError;
+use crate::elementext::ElementExt;
+
+pub fn generate(peripherals: &mut xmltree::Element, c: &chip::Chip) -> crate::Result<()> {
+    // Find the (first) peripheral with the name `CPU` to then add the interrupts to it.
+    for peripheral in peripherals.children.iter_mut() {
+        if peripheral.get_child("name".to_string()).and_then(|e| {
+            // Once `inner_deref` is stabilized, this can be changed to
+            // e.text.deref() or e.text.as_deref(), depending on the final name
+            e.text.as_ref().map(|s| s.as_ref())
+        }) == Some("CPU") {
+            peripheral.children.extend(c.interrupts.values().map(|interrupt| {
+                let mut el = xmltree::Element::new("interrupt");
+
+                el.children.push(xmltree::Element::new_with_text(
+                    "name",
+                    interrupt.name.clone(),
+                ));
+                el.children.push(xmltree::Element::new_with_text(
+                    "value",
+                    interrupt.index.to_string(),
+                ));
+
+                if let Some(ref desc) = interrupt.description {
```

Comment:
This will make `svd2rust` panic, because it **always** expects a description.  Instead, emit a warning and add a 'fake' description, like here:

https://github.com/Rahix/atdf2svd/blob/c7eb572c4ea12033bc2b9ccfe23d22ea2d859554/src/svd/field.rs#L9-L17